ui/src/Thing.elm

```diff
@@ -11,9 +13,9 @@ import Bootstrap.Modal as Modal
 import Bootstrap.Table as Table
 import Bootstrap.Utilities.Spacing as Spacing
 import Debug exposing (log)
-import Dict exposing (Dict)
+import Dict as Dict
```

Comment:
@darkodraskovic Why don't you just use `import Dict`? It will have the same effect, right?

Reply:
Fixed in the last commit.